server/src/main/java/com/cloud/usage/UsageServiceImpl.java

```diff
@@ -397,6 +363,59 @@ public boolean generateUsageRecords(GenerateUsageRecordsCmd cmd) {
         return new Pair<List<? extends Usage>, Integer>(usageRecords.first(), usageRecords.second());
     }
 
+    private Long getDomainScopeForQuery(ListUsageRecordsCmd cmd, Long accountId, Long domainId, Account caller, boolean isDomainAdmin) {
+        if (isDomainAdmin) {
```

Comment:
We could invert this statement and return `domainId` to reduce the indentation.

Reply:
@GutoVeronezi I like this better
the choice is
- two indentation levels and one return/exit point vs
- one indentation and two returns/exit points
I rather have the first.